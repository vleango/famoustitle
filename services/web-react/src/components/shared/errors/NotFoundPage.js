import React from 'react';
import {Helmet} from "react-helmet";

export default class NotFoundPage extends React.Component {

    render() {
        return (
            <div className='container'>
                <Helmet>
                    <title>Page not found - FamousTitle.com</title>
                </Helmet>

                404 - not found
            </div>
        );
    }

}
