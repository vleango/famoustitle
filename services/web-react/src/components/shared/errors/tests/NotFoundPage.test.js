import React from 'react';
import { shallow } from 'enzyme';

import NotFoundPage from '../NotFoundPage';

describe('Shared', () => {
    describe('Errors', () => {
        describe('NotFoundPage', () => {
            let wrapper = shallow(<NotFoundPage />);

            it('should correctly render NotFoundPage', () => {
                expect(wrapper).toMatchSnapshot();
            });
        });

    });
});
