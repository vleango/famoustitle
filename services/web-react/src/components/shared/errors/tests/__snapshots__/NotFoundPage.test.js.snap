// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shared Errors NotFoundPage should correctly render NotFoundPage 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFoundPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <HelmetWrapper
          defer={true}
          encodeSpecialCharacters={true}
>
          <title>
                    Page not found - FamousTitle.com
          </title>
</HelmetWrapper>,
        "404 - not found",
      ],
      "className": "container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <title>
            Page not found - FamousTitle.com
</title>,
          "defer": true,
          "encodeSpecialCharacters": true,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Page not found - FamousTitle.com",
          },
          "ref": null,
          "rendered": "Page not found - FamousTitle.com",
          "type": "title",
        },
        "type": [Function],
      },
      "404 - not found",
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <HelmetWrapper
            defer={true}
            encodeSpecialCharacters={true}
>
            <title>
                        Page not found - FamousTitle.com
            </title>
</HelmetWrapper>,
          "404 - not found",
        ],
        "className": "container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <title>
              Page not found - FamousTitle.com
</title>,
            "defer": true,
            "encodeSpecialCharacters": true,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Page not found - FamousTitle.com",
            },
            "ref": null,
            "rendered": "Page not found - FamousTitle.com",
            "type": "title",
          },
          "type": [Function],
        },
        "404 - not found",
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
